docs/framework/quickstart.mdx

```diff
@@ -14,63 +18,111 @@ Writer Framework works on Linux, Mac, and Windows. Python 3.9.2 or higher is req
   </Step>
 </Steps>
 
-## Create an app
+## Step 2: Create a new framework app
 
-You can use the command `writer create`, which will create a placeholder app in the path provided.
+Now that we've tested the setup, let's create our first "Hello, World" app using the `ai-starter` template. This template showcases how quickly you can build AI-powered applications with the Writer Framework.
 
-```sh
-writer create [path]
+Use the following command to generate the app structure:
 
-# Creates a new app in folder "testapp"
-writer create testapp
+```sh
+writer create hello-world --template=ai-starter
 ```
 
-A Framework app is a folder with the following items.
+This will create a new app folder, **hello-world**, with the following structure:
 
 1. `main.py` - The entry point for the app. You can import anything you need from here.
 2. `.wf/` - This folder contains the UI component declarations. Maintained by the Writer Framework's visual editor.
-3. `static/` - This folder contains front-end static files which you might want to distribute with your app. For example, images and stylesheets.
+3. `static/` - This folder contains front-end static files which you might want to distribute with your app, such as images and stylesheets.
 
-## Start the editor
+## Step 3: Start the visual editor  
 
-You can use the command `writer edit`.
+To start building and customizing your app visually, use the Writer Framework’s editor. This browser-based interface allows you to drag components, arrange layouts, and modify settings in real-time.
 
 ```sh
-writer edit [path]
-
-# Edit the application in subfolder "testapp"
-writer edit testapp
+writer edit hello-world
 ```
 
-This command will provide you with a local URL which you can use to access the Builder.
+After running this command, you'll see a local URL displayed in your command line. Open this URL in a web browser to access the editor.
+
+<Frame caption="Writer Framework editor opens at https://localhost:3006">
+  <img src="/framework/images/Quickstart/hello-world-editor.png" />
+</Frame>
+
+<Steps>
+  <Step title="Launch the Editor">
+    In your terminal, enter the following command to open the editor:
+    
+    ```sh
+    writer edit hello-world
+    ```
+
+    This command opens the Writer Framework editor in your browser, where you’ll see a live preview of your app. The editor interface includes a **component library** on the left, a **canvas** in the center for building layouts, and a **settings panel** on the right for customizing selected components.
+
+    <Info>
+The editor starts by default on port 3006. If you launch multiple editors in parallel and do not specify a port, Writer Framework will automatically assign the next port until reaching the limit of 4099.
```

Comment:
This port range isn't correct, the original statement is

> The editor starts by default on port 4005.

cc @FabienArcellier

Reply:
@ajot can we fix this? Happy to merge afterwards

Reply:
Maybe I am missing something, but for me, the editor starts at port 3006 by default, and the app runs at port 3005. Is that incorrect?

Reply:
@ajot I believe this changed in the latest RC, you should see it when you test Workflows (right @ramedina86?)

Reply:
The port policy has changed to allow the cli to find available port by itself : https://github.com/writer/writer-framework/issues/567. 

On release candidate for 0.8, the port for the run start at `3005`, the port for editor start at `4005`

Reply:
Updated: https://github.com/writer/writer-framework/pull/596/commits/5b766357398169cf1a344c433d856050182ec390